### ISSUE TITLE
Delete all Messages and Files for a Friend without deleting the Friend

### ISSUE THREAD

**Issue by zoff99** [zoff99/ToxAndroidRefImpl#386]
add a menu item to just delete messages and files for a friend, but keep the friend

**Comment by mmwelt** [comment]
Yes, this would be very nice. It's currently possible to delete the items one by one, but a single menu item to clear the conversation would be much more convenient.
